### DISCUSSION TITLE
How to create persistent icons and apps?

### DISCUSSION THREAD

**Discussion by TechMitten** [DustinBrett/daedalOS#465] [Q&A]
How can you make the desktop icons and install apps, etc persistent (so regardless of the browser/device opening the OS, it will display the same content. Same icons, customizations....) 

Thanks!

**Comment by DustinBrett** [comment]
If you want to host the files than you can clone the repo and add files into the `public` folder, then on build they become persistent. But besides that there is no way to persist changes between devices/browsers as all changes are stored locally in IndexedDb. It could be done in the future if there was a way to login to an online storage which restored the session and it's data upon login.

**Reply by TechMitten** [reply]
> If you want to host the files than you can clone the repo and add files into the `public` folder, then on build they become persistent. But besides that there is no way to persist changes between devices/browsers as all changes are stored locally in IndexedDb. It could be done in the future if there was a way to login to an online storage which restored the session and it's data upon login.

Thank you. I'm mostly interested in only creating shortcuts to existing applications and to websites. I was hoping to turn the OS into a public portfolio. I noticed your personal site can do this (with icons you've set). That's all I'm after really, simple shortcuts on the desktop.

**Reply by DustinBrett** [reply]
Ok sure, then indeed the way to do this is to add/modify the `public` folder (https://github.com/DustinBrett/daedalOS/tree/main/public).

**Comment by TechMitten** [comment]
Thank you and nice work on this project!


<https://aka.ms/AAb9ysg>
________________________________
